worlds/shivers/Rules.py

```diff
@@ -71,13 +76,21 @@ def first_nine_ixupi_capturable(state: CollectionState, player: int) -> bool:
         and metal_capturable(state, player)
 
 
-def all_skull_dials_available(state: CollectionState, player: int) -> bool:
-    return state.can_reach("Prehistoric", "Region", player) and state.can_reach("Tar River", "Region", player) \
-        and state.can_reach("Egypt", "Region", player) and state.can_reach("Burial", "Region", player) \
-        and state.can_reach("Gods Room", "Region", player) and state.can_reach("Werewolf", "Region", player)
+def all_skull_dials_set(state: CollectionState, player: int) -> bool:
+    return state.has("Set Skull Dial: Prehistoric", player) \
+        and state.has("Set Skull Dial: Tar River", player) \
+        and state.has("Set Skull Dial: Egypt", player) \
+        and state.has("Set Skull Dial: Burial", player) \
+        and state.has("Set Skull Dial: Gods Room", player) \
+        and state.has("Set Skull Dial: Werewolf", player)
 
 
-def get_rules_lookup(player: int):
+def completion_condition(state: CollectionState, player: int) -> bool:
+    years_since_sep_30_1980 = relativedelta(datetime.now(), datetime.fromisoformat("1980-09-30")).years
```

Comment:
Do you really need to calculate `years_since_sep_30_1980` every time the completion condition is checked?

Reply:
Moved to constants